Taarafo.Core/Brokers/Storages/StorageBroker.GroupPosts.cs

```diff
@@ -24,6 +24,18 @@ public async ValueTask<GroupPost> DeleteGroupPostAsync(GroupPost groupPost)
 
             await broker.SaveChangesAsync();
 
+            return groupPostEntityEntry.Entity;
+        }
```

Comment:
Empty line here

Reply:
Should i add an empty line? Please help me understand thanks.

Reply:
Yes, you did it correctly